.jenkins/pytorch/macos-test.sh

```diff
@@ -4,22 +4,9 @@
 # shellcheck source=./macos-common.sh
 source "$(dirname "${BASH_SOURCE[0]}")/macos-common.sh"
 
-if [ -z "${CI}" ]; then
-  rm -rf "${WORKSPACE_DIR}"/miniconda3/lib/python3.6/site-packages/torch*
-fi
-
-export CMAKE_PREFIX_PATH=${WORKSPACE_DIR}/miniconda3/
```

Comment:
Is this var def not needed? I wasn't guarded by the `if [ -z "${CI}" ]; then` like the other lines

Reply:
Yeah, this is definitely not needed anymore.  In all MacOS jobs, i.e. https://github.com/pytorch/pytorch/actions/runs/3739211153/jobs/6346399018, it's assigned to `export CMAKE_PREFIX_PATH=/miniconda3/` which is actually an invalid path.  And thing still works